__tests__/api.test.js

```diff
@@ -0,0 +1,33 @@
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+
+
+beforeEach(()=> seed(testData))
+
+afterAll(()=> db.end())
+
+
+
+describe('api/categories', ()=>{
+    test('responds with status code 200', ()=>{
+        return request(app).get('/api/categories').expect(200)
+    })
+    test('responds with an array full of categories with slug and description properties', ()=>{
+        return request(app).get('/api/categories').expect(200)
+        .then((result)=>{
+            // console.log(result)
+            expect(result.body[0]).toHaveProperty('slug')
+            expect(result.body[0]).toHaveProperty('description')
+        })
```

Comment:
it would be nice to have a test to make sure we are getting back the correct amount of categories